helper/restore_helper.go

```diff
@@ -226,19 +245,27 @@ func doRestoreAgent() error {
 		if i < len(oidWithBatchList)-*copyQueue {
 			nextOidWithBatch := oidWithBatchList[i+*copyQueue]
 			nextOid := nextOidWithBatch.oid
-			nextBatchNum := nextOidWithBatch.batch
-			nextPipeToCreate := fmt.Sprintf("%s_%d_%d", *pipeFile, nextOid, nextBatchNum)
-			logVerbose(fmt.Sprintf("Oid %d, Batch %d: Creating pipe %s\n", nextOid, nextBatchNum, nextPipeToCreate))
-			err := createPipe(nextPipeToCreate)
-			if err != nil {
-				logError(fmt.Sprintf("Oid %d, Batch %d: Failed to create pipe %s\n", nextOid, nextBatchNum, nextPipeToCreate))
-				// In the case this error is hit it means we have lost the
-				// ability to create pipes normally, so hard quit even if
-				// --on-error-continue is given
-				return err
+
+			if nextOid != skipOid {
+				nextBatchNum := nextOidWithBatch.batch
+				nextPipeToCreate := fmt.Sprintf("%s_%d_%d", *pipeFile, nextOid, nextBatchNum)
+				logVerbose(fmt.Sprintf("Oid %d, Batch %d: Creating pipe %s\n", nextOid, nextBatchNum, nextPipeToCreate))
+				err := createPipe(nextPipeToCreate)
+				if err != nil {
+					logError(fmt.Sprintf("Oid %d, Batch %d: Failed to create pipe %s\n", nextOid, nextBatchNum, nextPipeToCreate))
+					// In the case this error is hit it means we have lost the
+					// ability to create pipes normally, so hard quit even if
+					// --on-error-continue is given
+					return err
+				}
 			}
 		}
 
+		if tableOid == skipOid {
+			logVerbose(fmt.Sprintf("Oid %d, Batch %d: skip due to skip file\n", tableOid, batchNum))
+			goto LoopEnd
```

Comment:
why not
```suggestion
			continue
```

Reply:
This is to perform possible error handling which potentially might occur.